src/Files.App/ViewModels/UserControls/Previews/BasePreviewModel.cs

```diff
@@ -85,9 +85,9 @@ public virtual async Task LoadAsync()
 		public async virtual Task<List<FileProperty>> LoadPreviewAndDetailsAsync()
 		{
 			// Requesting sizes larger than 220 may result in a small thumbnail
-			var iconData = await FileThumbnailHelper.LoadIconWithoutOverlayAsync(Item.ItemPath, Constants.ShellIconSizes.Jumbo, false, false, false, false);
-			if (iconData is not null)
-				await MainWindow.Instance.DispatcherQueue.EnqueueOrInvokeAsync(async () => FileImage = await iconData.ToBitmapAsync());
+			var result = await FileThumbnailHelper.GetIconAsync(Item.ItemPath, Constants.ShellIconSizes.Jumbo, false, false, IconOptions.None);
```

Comment:
As above. I think this line should one more line break.

Reply:
I'm still working on the code in `GetIconAsync` so I'd rather wait until that's complete before putting time into the formatting.

Reply:
Sure.